package.json

```diff
@@ -11,7 +11,7 @@
     "test": "standard && hallmark && (nyc -s node test.js | faucet) && nyc report",
     "test-browsers": "airtap --coverage --verbose test.js",
     "test-browsers-local": "airtap --coverage -p local test.js",
-    "coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
Can you run `npm rm coveralls --save-dev`? We no longer need it. Thanks!

Reply:
Added in 96172d1